src/pages/Rockets.jsx

```diff
@@ -24,6 +24,7 @@ export default function Rockets() {
             name={rocket.name}
             description={rocket.description}
             image={rocket.image[0]}
```

Comment:
The image for the first rocket is not loading, kindly fix that.
![Space Travellers' App and 45 more pages - Personal - Microsoft​ Edge 5_18_2023 9_40_01 PM](https://github.com/kalbek/react-group-project/assets/44664009/e116e454-84c5-4d09-b254-d88f6cbbfafd)

Reply:
Thank you for the suggestion, not sure why that's happening because all images are showing on my computer. 
![Capture react img](https://github.com/kalbek/react-group-project/assets/110445380/e5779410-2df3-4dfd-b8fa-def189fc1202)